packages/fuels/tests/harness.rs

```diff
@@ -2588,24 +2597,24 @@ async fn contract_call_fee_estimation() -> Result<(), Error> {
     let contract_instance = MyContractBuilder::new(contract_id.to_string(), wallet).build();
 
     let tolerance = 0.2;
-    let transaction_cost = contract_instance
+    let estimated_transaction_cost = contract_instance
         .initialize_counter(42) // Build the ABI call
         .tx_params(TxParameters::new(
             Some(10_000),
             Some(500),
             Some(10_000),
             None,
         ))
-        .get_transaction_cost(Some(tolerance)) // Perform the network call
+        .estimate_transaction_cost(Some(tolerance)) // Perform the network call
         .await?;
 
-    assert_eq!(transaction_cost.min_gas_price, 0);
-    assert_eq!(transaction_cost.min_byte_price, 0);
-    assert_eq!(transaction_cost.gas_price, 10_000);
-    assert_eq!(transaction_cost.byte_price, 10_000);
-    assert_eq!(transaction_cost.gas_used, 348);
-    assert_eq!(transaction_cost.byte_size, 704);
-    assert_eq!(transaction_cost.fee, 0.01052);
+    assert_eq!(estimated_transaction_cost.min_gas_price, 0);
+    assert_eq!(estimated_transaction_cost.min_byte_price, 0);
+    assert_eq!(estimated_transaction_cost.gas_price, 10_000);
+    assert_eq!(estimated_transaction_cost.byte_price, 10_000);
+    assert_eq!(estimated_transaction_cost.gas_used, 348);
+    assert_eq!(estimated_transaction_cost.byte_size, 704);
+    assert_eq!(estimated_transaction_cost.total_fee, 0.01052);
```

Comment:
Same remark here: we are not checking the math here, only the fact that it equals to some value. We should test that the math is the one we expect.